packages/elements/src/react/sign-in/passkey.tsx

```diff
@@ -0,0 +1,46 @@
+import { Slot } from '@radix-ui/react-slot';
+import * as React from 'react';
+
+import { SignInRouterCtx } from '~/react/sign-in/context';
+
+export type SignInPasskeyElement = React.ElementRef<'button'>;
+
+export type SignInPasskeyProps = {
+  asChild?: boolean;
+  children: React.ReactNode;
+} & React.DetailedHTMLProps<React.HTMLAttributes<HTMLButtonElement>, HTMLButtonElement>;
+
+const SIGN_IN_PASSKEY_NAME = 'SignInPasskey';
+
+/**
+ * Prompt users to select a passkey from their device in order to sign in.
+ * This component must be used within the <Step name="start">.
+ *
+ *
+ * @example
+ * import { Passkey } from '@clerk/elements/sign-in';
+ * <Passkey>Use passkey instead</Passkey>;
```

Comment:
We're leaving out the imports in those examples.
And since we want people to use namespaces we should use `<SignIn.Passkey>` here instead

```suggestion
 * <SignIn.Passkey>Use passkey instead</SignIn.Passkey>;
```